content/millennium/r4/base/individuals/related-person.md

```diff
@@ -214,15 +214,14 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Patch
 
-Patch an existing Encounter-level RelatedPerson.
+Patch an existing Encounter-level or Person-level RelatedPerson.
```

Comment:
https://github.com/cerner/fhir.cerner.com/pull/686/files#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584L225 here the line you deleted has patient level and we have named it as person level is it fine @SantanuBanerjee

Reply:
https://jira2.cerner.com/browse/OPENSVC-20543
have made the changes as per mentioned in the jira description by Alex

Reply:
I think it is fine since this note was added in the jira by Alex.